Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -118,6 +121,39 @@ public async ValueTask<IEnumerable<TItem>> UpdateAsync(IEnumerable<TItem> values
             return enumerable;
         }
 
+        /// <inheritdoc/>
+        public async ValueTask UpdateAsync(string id,
+            Action<IPatchOperationBuilder<TItem>> builder,
+            string partitionKeyValue = null,
+            CancellationToken cancellationToken = default)
+        {
+            await Task.CompletedTask;
+
+            partitionKeyValue ??= id;
+
+            IItem item = Items.Values.FirstOrDefault(x => x.Id == id && x.PartitionKey == partitionKeyValue);
+
+            if (item is null)
+            {
+                NotFound();
+            }
```

Comment:
```suggestion
            IItem item = Items.Values.FirstOrDefault(x => x.Id == id && x.PartitionKey == partitionKeyValue);
            if (item is null)
            {
                NotFound();
            }
```
When assigning a variable, it's preferred to remove the newline immediately following it when it's used straight away.